.github/workflows/lint.yml

```
@@ -33,7 +33,7 @@ jobs:
 
       # Runs the Super-Linter action
       - name: Run Super-Linter on all codebase
-        uses: github/super-linter@v3
+        uses: github/super-linter@slim-v4
```

```suggestion
        uses: ghcr.io://github/super-linter@slim-v4
```
That's what's written in the documentation ;)

---

Not sure why their doc is using this only for the slim, it's a bit confusing. Thanks for spotting this!